solutions/pytest_folder/test_parametrization.py

```diff
@@ -0,0 +1,81 @@
+# Задание содержится в tasks/test_parametrization.md
+
+import math
+import time
+
+import pytest
+from selenium.common import TimeoutException
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as ec
+from selenium.webdriver.support.ui import WebDriverWait
+
+from helpers import check_if_user_is_authorized
+
+urls = [
+    "236895",
+    "236896",
+    "236897",
+    "236898",
+    "236899",
+    "236903",
+    "236904",
+    "236905",
+]
+
+
+@pytest.mark.parametrize("link", urls)
+def test_check_urls_on_feedback(browser, auth_data, link):
+    """Test to check feedback text after successfully solving task."""
+    # авторизация
+    browser.get(f"https://stepik.org/lesson/{link}/step/1")
+
+    enter_button = browser.find_element(By.CLASS_NAME, "navbar__auth_login ")
+    enter_button.click()
+
+    input_login = browser.find_element(By.ID, "id_login_email")
+    input_login.send_keys(auth_data["login"])
+
+    input_password = browser.find_element(By.ID, "id_login_password")
+    input_password.send_keys(auth_data["password"])
+
+    submit_button = browser.find_element(By.CLASS_NAME, "sign-form__btn")
+    submit_button.click()
+
+    # проверка, авторизован ли пользователь
+    user_is_authorized = check_if_user_is_authorized(browser)
```

Comment:
```suggestion
    is_user_authorized = check_if_user_is_authorized(browser)
```

Или, согласно [этой статье](https://michaelzanggl.com/articles/tips-on-naming-boolean-variables/)...
```python
    is_user_logged_in = check_if_user_is_authorized(browser)
```
На мой взгляд, так проще читается.

Reply:
там больше не нужна эта переменная, пока на паузу. Статью прочла